clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -2923,14 +2923,20 @@ static void handleWorkGroupSize(Sema &S, Decl *D, const ParsedAttr &AL) {
     return;
 
   uint32_t WGSize[3];
+  if (AL.getAttributeSpellingListIndex() == 2) {
```

Comment:
There is an enum class that gets created in the attribute type itself that has these lists.  Something like ReqdWorkGroupSizeAttr::CXX11_intel_reqd_work_group_size

Reply:
Got it.

Reply:
There is no ReqdWorkGroupSizeAttr::CXX11_intel_reqd_work_group_size enum, and I don't know how I can find valid enum.

Reply:
I cannot see the exact spelling because I don't have your patch installed, so i sort of guessed based on the spelling.  Look in your build directory at:
whatever/build/tools/clang/include/clang/Sema/Attrs.inc
You'll see something like this:

     class SectionAttr : public InheritableAttr {
     unsigned nameLength;
     char *name;
    
     public:
       enum Spelling {
          GNU_section = 0,
          CXX11_gnu_section = 1,
          Declspec_allocate = 2,
        SpellingNotCalculated = 15
        };
        ...

See the Spelling enum which should have the entry for your new spelling.

Reply:
I watched in this file and I didn't find enum Spelling in class ReqdWorkGroupSize. It is very strange.